pkg/drivers/powerflex.go

```diff
@@ -247,11 +257,16 @@ func ModifyPowerflexCR(yamlString string, cr csmv1.ContainerStorageModule, fileT
 			if env.Name == "X_CSI_QUOTA_ENABLED" {
 				enableQuota = env.Value
 			}
+			if env.Name == "X_CSI_HEALTH_MONITOR_ENABLED" {
+				healthMonitorController = env.Value
```

Comment:
i guess this should be healthMonitorNode since its under node

Reply:
1) Please check the logs from node pods to verify if its working on node side 
2) describe csm object and verify if the node environment variable was configured